elfo-configurer/src/lib.rs

```diff
@@ -255,15 +253,65 @@ impl Configurer {
             Err(errors)
         }
     }
+
+    async fn update_all(
+        &self,
+        config_list: &[ConfigWithMeta],
+    ) -> Result<(), Vec<ReloadConfigsError>> {
+        let futures = config_list
+            .iter()
+            .cloned()
+            .map(|item| {
+                let group = item.group_name;
+                // While `UpdateConfig` is defined as a request to cover more use cases, default
+                // configurer simply sends out new configs instead of waiting for all groups to
+                // process the message and respond. This is done for performance reasons. If an
+                // actor has a congested mailbox or spends a lot of time processing each
+                // message, updating configs using requests can take a lot of
+                // time.
+                let fut = self.ctx.send_to(item.addr, UpdateConfig::new(item.config));
+
+                wrap_long_running_future(
+                    fut,
+                    group,
+                    "some actors in the group have congested mailboxes, config update is stalled",
+                )
+            })
+            .collect::<Vec<_>>();
+
+        let errors = future::join_all(futures)
+            .await
+            .into_iter()
+            .filter_map(|(group, result)| match result {
+                Ok(_) => None,
+                Err(err) => {
```

Comment:
Should we ignore errors altogether here, like the current code does? It is valid for the actor to not accept any config updates at all.